[2024-11-14T22:00:20.600+0700] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-14T22:00:20.658+0700] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_vlance_dag.update_vlance_jobs manual__2024-11-14T15:00:18.446812+00:00 [queued]>
[2024-11-14T22:00:20.662+0700] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_vlance_dag.update_vlance_jobs manual__2024-11-14T15:00:18.446812+00:00 [queued]>
[2024-11-14T22:00:20.663+0700] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-14T22:00:20.672+0700] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): update_vlance_jobs> on 2024-11-14 15:00:18.446812+00:00
[2024-11-14T22:00:20.675+0700] {standard_task_runner.py:72} INFO - Started process 170041 to run task
[2024-11-14T22:00:20.677+0700] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_vlance_dag', 'update_vlance_jobs', 'manual__2024-11-14T15:00:18.446812+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/ETL_vlance.py', '--cfg-path', '/tmp/tmp57m9ohdn']
[2024-11-14T22:00:20.678+0700] {standard_task_runner.py:105} INFO - Job 25: Subtask update_vlance_jobs
[2024-11-14T22:00:20.698+0700] {task_command.py:467} INFO - Running <TaskInstance: etl_vlance_dag.update_vlance_jobs manual__2024-11-14T15:00:18.446812+00:00 [running]> on host fedora
[2024-11-14T22:00:20.739+0700] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='etl_vlance_dag' AIRFLOW_CTX_TASK_ID='update_vlance_jobs' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T15:00:18.446812+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-14T15:00:18.446812+00:00'
[2024-11-14T22:00:20.740+0700] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-14T22:00:20.749+0700] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-14T22:00:20.749+0700] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/bin/python /home/nngiaminh1812/airflow/dags/vlance_update.py']
[2024-11-14T22:00:20.754+0700] {subprocess.py:99} INFO - Output:
[2024-11-14T22:09:31.381+0700] {subprocess.py:106} INFO - Solve captcha manually...............
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - Page: 1
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - 4
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - /viec-freelance/can-xay-dung-trang-web-ban-hang?utm_source=homepage&utm_medium=job-list-block&utm_campaign=20210420_homepage_job-list-block&utm_content=link
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - Crawled ID is smaller than or equal the latest existed one
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - /viec-freelance/du-an-outsourcing-1?utm_source=homepage&utm_medium=job-list-block&utm_campaign=20210420_homepage_job-list-block&utm_content=link
[2024-11-14T22:09:31.382+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - /viec-freelance/content-writer-viet-noi-dung?utm_source=homepage&utm_medium=job-list-block&utm_campaign=20210420_homepage_job-list-block&utm_content=link
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - /cd-tang-review/review-khoa-hoc-tren-udemy-12
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - Chiến dịch hiển thị
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - Page: 2
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - 10
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - /viec-freelance/lap-trinh-mach-stm32-1
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.383+0700] {subprocess.py:106} INFO - /viec-freelance/seo-website-game-bai-bong-da-1
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO - /viec-freelance/cai-dat-server-ddns-va-cau-hinh-he-thong-ddns
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO - /viec-freelance/decompile-file-lua
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO - Đăng việc
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO - Traceback (most recent call last):
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO -   File "/home/nngiaminh1812/airflow/dags/vlance_update.py", line 305, in <module>
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO -     update_vlance_jobs("qn9965mh@gmail.com","123456asd*")
[2024-11-14T22:09:31.384+0700] {subprocess.py:106} INFO -   File "/home/nngiaminh1812/airflow/dags/vlance_update.py", line 174, in update_vlance_jobs
[2024-11-14T22:09:31.385+0700] {subprocess.py:106} INFO -     bid_counter = soup.find('span', attrs={'class':"bid-counter"}).text
[2024-11-14T22:09:31.385+0700] {subprocess.py:106} INFO -                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-11-14T22:09:31.385+0700] {subprocess.py:106} INFO - AttributeError: 'NoneType' object has no attribute 'text'
[2024-11-14T22:09:31.721+0700] {subprocess.py:110} INFO - Command exited with return code 1
[2024-11-14T22:09:31.726+0700] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-11-14T22:09:31.730+0700] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_vlance_dag, task_id=update_vlance_jobs, run_id=manual__2024-11-14T15:00:18.446812+00:00, execution_date=20241114T150018, start_date=20241114T150020, end_date=20241114T150931
[2024-11-14T22:09:31.779+0700] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-14T22:09:31.780+0700] {standard_task_runner.py:124} ERROR - Failed to execute job 25 for task update_vlance_jobs (Bash command failed. The command returned a non-zero exit code 1.; 170041)
Traceback (most recent call last):
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nngiaminh1812/.pyenv/versions/3.11.0/envs/airflow_env/lib/python3.11/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-11-14T22:09:31.806+0700] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-14T22:09:31.831+0700] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T22:09:31.832+0700] {local_task_job_runner.py:245} INFO - ::endgroup::
